contracts/Core.sol

```diff
@@ -186,7 +186,13 @@ contract Core is CoreInterface, Util {
 	/**
 	 *	@notice Verify account proof of OpenSTRemote and commit storage root at given block height
 	 *
-	 *  @dev ProofVerificationSkipped event needed to identify replay calls for same block height
+	 *  @dev proveOpenST can be called by anyone to verify merkle proof of OpenSTRemote contract address. OpenSTRemote is OpenSTUtility
+	 *		 contract address on utility chain and OpenSTValue contract address on value chain.
+	 *		 Trust factor is brought by stateRoots mapping. stateRoot is committed in commitStateRoot function by mosaic process
+	 *		 which is a trusted decentralized system running separately.
+	 * 		 It's important to note that in replay calls of proveOpenST bytes _rlpParentNodes variable is not validated. In this case
+	 *		 input storage root is verified with stored storage root of the first method call.
```

Comment:
Somewhat similarly to my question above, the way I read this function, I feel like "input storage root is verified with stored storage root" is somewhat misleading, because the "input storage root" is derived from the `_rlpEncodedAccount` and "verified" is really just an equality check.

Reply:
You are right. Updated the comment like below

"input storage root derived from merkle proof account nodes is verified with stored storage root of given blockHeight."

Let me know if it's fine.